src/allotropy/allotrope/schema_mappers/adm/plate_reader/rec/_2024/_06/plate_reader.py

```diff
@@ -403,3 +510,28 @@ def _get_calculated_data_aggregate_document(
                 for calculated_data_item in calculated_data_items
             ]
         )
+
+    def _get_data_cube(self, profile_data_cube: DataCube) -> TDatacube | None:
+        return TDatacube(
+            label=str(profile_data_cube.label),
+            cube_structure=TDatacubeStructure(
+                dimensions=[
+                    TDatacubeComponent(
+                        concept=profile_data_cube.structure_dimensions.concept,
+                        field_componentDatatype=profile_data_cube.structure_dimensions.type_,
+                        unit=profile_data_cube.structure_dimensions.unit,
+                    )
+                ],
+                measures=[
+                    TDatacubeComponent(
+                        field_componentDatatype=profile_data_cube.structure_measures.type_,
+                        concept=profile_data_cube.structure_measures.concept,
+                        unit=profile_data_cube.structure_measures.unit,
+                    )
+                ],
+            ),
+            data=TDatacubeData(
+                dimensions=[profile_data_cube.dimensions],
+                measures=[profile_data_cube.measures],
```

Comment:
these two are already lists.
```suggestion
                dimensions=profile_data_cube.dimensions,
                measures=profile_data_cube.measures,
```